src/libraries/Directory.Build.targets

```diff
@@ -126,6 +126,9 @@
                                                 '$(IsTestProject)' != 'true' and
                                                 '$(IsTestSupportProject)' != 'true' and
                                                 '$(IsGeneratorProject)' != 'true'">true</SkipTargetingPackShimReferences>
+
+    <!-- Make libraries packages use and require a package readme file. -->
+    <EnableDefaultPackageReadmeFile Condition="'$(EnableDefaultPackageReadmeFile)' == '' and '$(IsShipping)' == 'true'">true</EnableDefaultPackageReadmeFile>
```

Comment:
In which CI leg do we enable IsShipping?

Reply:
IsShipping defaults to true and some projects (ie transport ones) set it to false.